tests/queries/0_stateless/01171_mv_select_insert_isolation_long.sh

```diff
@@ -110,36 +153,69 @@ function thread_select_insert()
         SELECT throwIf(1 != (SELECT countDistinct(arr) FROM (SELECT x, arraySort(groupArray(nm)) AS arr FROM tmp WHERE x!=4 GROUP BY x))) FORMAT Null;
         SELECT throwIf((SELECT count(), sum(nm) FROM tmp WHERE x=4) != (SELECT count(), sum(nm) FROM tmp WHERE x!=4)) FORMAT Null;
         ROLLBACK;"
-    done
 }
 
-thread_insert_commit 1 & PID_1=$!
-thread_insert_commit 2 & PID_2=$!
-thread_insert_rollback 3 & PID_3=$!
-
-thread_optimize & PID_4=$!
-thread_select & PID_5=$!
-thread_select_insert & PID_6=$!
-sleep 0.$RANDOM;
-thread_select & PID_7=$!
-thread_select_insert & PID_8=$!
-
-wait $PID_1 && wait $PID_2 && wait $PID_3
-kill -TERM $PID_4
-kill -TERM $PID_5
-kill -TERM $PID_6
-kill -TERM $PID_7
-kill -TERM $PID_8
-wait
-wait_for_queries_to_finish 40
+MAIN_TIME_PART=400
```

Comment:
This has made the test 40x slower in release build (used to take 10 seconds, now it takes 400): https://play.clickhouse.com/play?user=play#U2VsZWN0ICogZnJvbSBjaGVja3Mgd2hlcmUgdGVzdF9uYW1lID0gJzAxMTcxX212X3NlbGVjdF9pbnNlcnRfaXNvbGF0aW9uX2xvbmcnIEFORCBjaGVja19uYW1lID0gJ1N0YXRlbGVzcyB0ZXN0cyAocmVsZWFzZSknIGFuZCBwdWxsX3JlcXVlc3RfbnVtYmVyID0gMCBvcmRlciBieSBjaGVja19zdGFydF90aW1lIGRlc2M=

And it's executed in sequential mode. So it's adding 6 minutes to all runs.

Reply:
Lest remove `no-parallel`. It should be fine.
I will do this.

Reply:
https://github.com/ClickHouse/ClickHouse/pull/67470